tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "buildOnSave": false,
   "compilerOptions": {
     "module": "commonjs",
```

Comment:
Setting `"module": "es6" ` breaks the AoT compilation, it leads to errors like `Cannot find module '@angular/router'`. And if set `"target"` also to `"es6"`, `Duplicate identifier 'PropertyKey'` error arises.